test/groovy/CloudFoundryDeployTest.groovy

```diff
@@ -330,10 +330,15 @@ class CloudFoundryDeployTest extends BasePiperTest {
 
         assertThat(jscr.shell, hasItem(containsString('cf login -u "test_cf" -p \'********\' -a https://api.cf.eu10.hana.ondemand.com -o "testOrg" -s "testSpace"')))
         assertThat(jscr.shell, hasItem(containsString("cf blue-green-deploy testAppName -f 'test.yml'")))
-        assertThat(jscr.shell, hasItem(containsString("cf stop testAppName-old")))
+        assertThat(jscr.shell, hasItem(containsString("cf stop testAppName-old > cfStopOutput.txt")))
         assertThat(jscr.shell, hasItem(containsString("cf logout")))
     }
 
+    @Test
+    void testCfNativeBlueGreenKeepOldInstanceShouldFail(){
+        assert true
```

Comment:
?

Reply:
Still WIP. I will implement the test later today.